sdk/loadtestservice/azure-developer-loadtesting/src/main/java/com/azure/developer/loadtesting/LoadTestAdministrationAsyncClient.java

```diff
@@ -38,314 +43,322 @@ public final class LoadTestAdministrationAsyncClient {
+     *             metricNamespace: String (Required)
      *             displayDescription: String (Optional)
-     *             name (Required): {
-     *                 value: String (Required)
-     *                 localizedValue: String (Required)
-     *             }
+     *             name: String (Required)
      *             aggregation: String (Required)
      *             unit: String (Optional)
      *             resourceType: String (Required)
      *         }
      *     }
+     *     createdDateTime: OffsetDateTime (Optional)
+     *     createdBy: String (Optional)
+     *     lastModifiedDateTime: OffsetDateTime (Optional)
+     *     lastModifiedBy: String (Optional)
      * }
      * }</pre>
      *
-     * @param name Unique name for server metrics, must be a valid URL character ^[a-z0-9_-]*$.
+     * @param testId Unique name for the load test, must contain only lower-case alphabetic, numeric, underscore or
+     *     hyphen characters.
      * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
      * @throws HttpResponseException thrown if the request is rejected by server.
      * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
      * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
      * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
-     * @return server metrics configuration by its name along with {@link Response} on successful completion of {@link
-     *     Mono}.
+     * @return server metric configuration for the given test along with {@link Response} on successful completion of
+     *     {@link Mono}.
      */
     @Generated
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<BinaryData>> getServerMetricsByNameWithResponse(String name, RequestOptions requestOptions) {
-        return this.serviceClient.getServerMetricsByNameWithResponseAsync(name, requestOptions);
+    public Mono<Response<BinaryData>> getServerMetricsConfigWithResponse(String testId, RequestOptions requestOptions) {
+        return this.serviceClient.getServerMetricsConfigWithResponseAsync(testId, requestOptions);
+    }
+
+    /** Validation status of uploaded file */
+    public enum ValidationStatus {
+
+        /** Validation not initiated */
+        NOT_VALIDATED,
+        /** Validation successful */
+        VALIDATION_SUCCESS,
+        /** Validation failed */
+        VALIDATION_FAILURE,
+        /** Validation initiated */
+        VALIDATION_INITIATED,
+        /** Validation not required for file type */
+        VALIDATION_NOT_REQUIRED
+    }
+
+    private Mono<PollResponse<ValidationStatus>> getValidationStatus(BinaryData fileMono) {
+        String validationStatus;
+        JsonNode file;
+        try {
+            file = new ObjectMapper().readTree(fileMono.toString());
```

Comment:
`ObjectMapper` should be a static class variable, to be shared (it is thread-safe).